src/crimson/os/seastore/segment_manager.h

```diff
@@ -16,6 +17,60 @@
 
 namespace crimson::os::seastore {
 
+using magic_t = uint64_t;
+
+using device_spec_t =
+  std::tuple<magic_t, device_type_t, device_id_t>;
```

Comment:
Eh... let's make this a proper struct with member names and an encode/decode with versioning.  It's ondisk configuration state in a cold path, so we may want to expand this in the future and we don't care about encode/decode performance.

Reply:
will do:-)